vault/resource_ldap_auth_backend.go

```diff
@@ -204,6 +209,10 @@ func ldapAuthBackendUpdate(d *schema.ResourceData, meta interface{}) error {
 		data["bindpass"] = v.(string)
 	}
 
+	if v, ok := d.GetOk("case_sensitive_names"); ok {
```

Comment:
I _think_ this needs to be `d.GetOkExists` like `starttls` above, otherwise a value of `false` wont return `ok == true`, so we could be in a situation where we can enable this but never disable it. A default value seems in order as well, if possible

Reply:
I just created this mr with an idea this might solve it. I probably need some help with it. And how to test it. But will get back to this on Monday.

Reply:
```suggestion
	if v, ok := d.GetOkExists("case_sensitive_names"); ok {
```